### PR TITLE
Web UI: Fix strings

### PR DESCRIPTION
**Pull request by poncovka**
* Fix the capital letter in `Installer`.
* Replace the `Storage configuration` title.

### CODE REVIEW

File: ui/webui/src/components/storage/InstallationDestination.jsx [hunk 1]
```diff
@@ -158,7 +158,7 @@ const LocalStandardDisks = ({ idPrefix, onAddErrorNotification }) => {
                   id="installation-destination-table-label"
                 >
                     {cockpit.format(
-                        cockpit.ngettext("$0 (of $1) disk selected", "$0 (of $1) disks selected", selectedDisksCnt),
+                        "$0 (of $1) disks selected",
```
Comment (VladimirSlavik) [inline ui/webui/src/components/storage/InstallationDestination.jsx:161]: This is incorrect in most languages. See https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html after "The following rules are known at this point.".

**Review by VladimirSlavik — CHANGES_REQUESTED** [review]
Lower i is fine, the plural form is an English-only change.

File: ui/webui/src/components/storage/InstallationDestination.jsx (hunk 1 above)
Comment (VladimirSlavik) [reply ui/webui/src/components/storage/InstallationDestination.jsx:161]: FWIW, this is a design problem, not a PR problem, really.

**Comment by poncovka** [comment]
Updated.

**Review by VladimirSlavik — APPROVED** [review]
Thank you, LGTM.

You will have to change the names also in the tests, but that's expected.

**Review by jkonecny12 — APPROVED** [review]
Looks good to me.

**Comment by lclay2** [comment]
@poncovka , I replied to your question on the slide deck.

**Pull request merged**